taxonomies/categories.txt

```diff
@@ -44189,10 +44979,12 @@ es:Garbanzos de Escacena
 
 <en:Pulses
 en:Common beans, String beans
+ca:Fesols
 de:Gartenbohnen, Grüne Bohnen, Fisolen, Strankerl, Stangenbohnen
```

Comment:
`ERROR - de:grüne-bohnen already is a synonym of grüne-bohnen (en:green-beans) - cannot make a synonym of gartenbohnen for en:common-beans`

Reply:
- @stephanegigandet